akka-http-cors/src/main/resources/reference.conf

```diff
@@ -0,0 +1,76 @@
+    #
+    # Default: `true`
+    #
+    # @see [[https://www.w3.org/TR/cors/#access-control-allow-credentials-response-header Access-Control-Allow-Credentials]]
+    allow-credentials = true
+
+    # List of origins that the CORS filter must allow. Can also be set to `*` to allow
+    # access to the resource from any origin. Controls the content of the
+    # `Access-Control-Allow-Origin` response header: if parameter is `*` and credentials
+    # are not allowed, a `*` is set in `Access-Control-Allow-Origin`. Otherwise, the
+    # origins given in the `Origin` request header are echoed.
+    #
+    # The actual or preflight request is rejected if any of the origins from the request
+    # is not allowed.
+    #
+    # Default: `*`
+    #
+    # @see [[https://www.w3.org/TR/cors/#access-control-allow-origin-response-header Access-Control-Allow-Origin]]
+    allowed-origins = ["*"]
+
+    # List of request headers that can be used when making an actual request. Controls
+    # the content of the `Access-Control-Allow-Headers` header in a preflight response:
+    # if parameter is `*`, the headers from `Access-Control-Request-Headers` are echoed.
+    # Otherwise the parameter list is returned as part of the header.
+    #
+    # Default: `HttpHeaderRange.*`
+    #
+    # @see [[https://www.w3.org/TR/cors/#access-control-allow-headers-response-header Access-Control-Allow-Headers]]
+    allowed-headers = ["*"]
+
+    # List of methods that can be used when making an actual request. The list is
+    # returned as part of the `Access-Control-Allow-Methods` preflight response header.
+    #
+    # The preflight request will be rejected if the `Access-Control-Request-Method`
+    # header's method is not part of the list.
+    #
+    # Default: `Seq(GET, POST, HEAD, OPTIONS)`
+    #
+    # @see [[https://www.w3.org/TR/cors/#access-control-allow-methods-response-header Access-Control-Allow-Methods]]
+    allowed-methods = ["GET", "POST", "HEAD", "OPTIONS"]
+
+    # List of headers (other than simple response headers) that browsers are allowed to access.
+    # If not empty, this list is returned as part of the `Access-Control-Expose-Headers`
+    # header in the actual response.
+    #
+    # Default: `Seq.empty`
+    #
+    # @see [[https://www.w3.org/TR/cors/#simple-response-header Simple response headers]]
+    # @see [[https://www.w3.org/TR/cors/#access-control-expose-headers-response-header Access-Control-Expose-Headers]]
+    exposed-headers = []
+
+    # When set, the amount of seconds the browser is allowed to cache the results of a preflight request.
+    # This value is returned as part of the `Access-Control-Max-Age` preflight response header.
+    # If `None`, the header is not added to the preflight response.
+    #
+    # Default: `Some(30 * 60)`
+    #
+    # @see [[https://www.w3.org/TR/cors/#access-control-max-age-response-header Access-Control-Max-Age]]
+    max-age = 1800
```

Comment:
The syntax "30 minutes" should also be supported: 
- If it's `null`, interpret it as `None`;
- If it's a duration, convert it to seconds;
- If it's a number, interpret it as seconds.